tests/common.py

```diff
@@ -1,7 +1,14 @@
-from stereotype import Model
+from stereotype import Model, StrField, DEFAULT_ROLE
 
 
 class Leaf(Model):
     type = 'leaf'
 
     color: str = 'green'
+
+
+class PrivateStrField(StrField):
+    def to_primitive(self, value, role=DEFAULT_ROLE, context=None):
+        if context is not None and context.get('private', False):
+            return '<hidden>'
+        return value
```

Comment:
I wouldn't even point out this nitpick if no other changes were needed, but would be nice to ensure the standard trailing newline. :)